### PR TITLE
5.1 changes

### PR DESCRIPTION
**Pull request by vdelendik**

### CODE REVIEW

File: manual_deployment/configScriptApproval.groovy [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import org.jenkinsci.plugins.scriptsecurity.scripts.*;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.*;
```
Comment (vdelendik) [inline manual_deployment/configScriptApproval.groovy:2]: @ilya-varchenya 
let's remove "manual_deployment/configScriptApproval.groovy" file. on 3rd party jenkins instances they can apply those changes via jenkins UI when necessary

Comment (vdelendik) [reply manual_deployment/configScriptApproval.groovy]: done on mine own

**Comment by vdelendik** [comment]
:+1:

**Comment by sandinosanchez** [comment]
👍

**Comment by ilya-varchenya** [comment]
👍

**Pull request merged**